doc/interface.md

```diff
@@ -19,6 +19,11 @@ obsolete and is not recommended to use.
 Rust interface contributed by Seaton Ullberg. The documentation is
 found at <https://docs.rs/crate/spglib/>.
 
+## Julia interface
+
+Julia interface contributed by [Qi Zhang](https://github.com/singularitti). The documentation is
```

Comment:
Can you link to the github repository or the package manager? Not sure about Julia packaging.

Reply:
Sure. My repo is here: https://github.com/singularitti/Spglib.jl.
But I put the link to JuliaHub: https://juliahub.com/ui/Packages/General/Spglib,
it is kind of like Python's PyPI.